app/assets/javascripts/modules/orgs.js

```diff
@@ -13,6 +13,22 @@ angular.module('probedock.orgs', [ 'probedock.api', 'probedock.auth', 'probedock
         service.emit('refresh', service.organizations);
       },
 
+      getOrganization: function(orgId) {
+        var org =_.findWhere(service.organizations, { id: orgId });
+
+        if (_.isUndefined(org)) {
+          return api({
+            url: '/organizations/' + orgId
+          })
+          .then(function(response) {
```

Comment:
I prefer it that way, but I don't care one way or another, as long as we define a clear convention and apply it to all new and updated code.